src/lib/OpenEXR/ImfCompression.h.in

```diff
@@ -0,0 +1,61 @@
+//
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) Contributors to the OpenEXR Project.
+//
+
+#ifndef INCLUDED_IMF_COMPRESSION_H
+#define INCLUDED_IMF_COMPRESSION_H
+
+//-----------------------------------------------------------------------------
+//
+//	enum Compression
+//
+//-----------------------------------------------------------------------------
+#include "ImfForward.h"
+#include <string>
+
+OPENEXR_IMF_INTERNAL_NAMESPACE_HEADER_ENTER
+
+enum IMF_EXPORT_ENUM Compression
+{
+    // DO NOT EDIT: THIS FILE IS GENERATED BY CMAKE.
```

Comment:
Would it be overkill to say something like `DO NOT EDIT: ImfCompression.h IS GENERATED FROM ImfCompression.h.in BY CMAKE (with UPDATE_COMPRESSOR_LIST=ON)` so that anyone tempted to edit anything in ImfCompression.h knows exactly what to change instead and how to get changes through, and also moving the comment so it's more obvious at the top of the file?

Anyone wanting to make any change anywhere in ImfCompression.h needs to edit ImfCompression.h.in instead and cmake run again with UPDATE_COMPRESSOR_LIST=ON, even if the change won't change the Compression enum, otherwise the change will be lost.

Perhaps we should also make sure the CI runs with UPDATE_COMPRESSOR_LIST=ON to help catch issues if people edit the wrong file.

Reply:
all comments were addressed. Thanks for the feedback !